src/dsp_tools/excel2xml.py

```diff
@@ -1267,14 +1267,14 @@ def make_text_prop(
     for val in values:
         kwargs = {"permissions": val.permissions}
         if check_notna(val.comment):
-            kwargs["comment"] = val.comment  # type: ignore
+            kwargs["comment"] = val.comment  # type: ignore[assignment]
         if check_notna(val.encoding):
-            kwargs["encoding"] = val.encoding  # type: ignore
+            kwargs["encoding"] = val.encoding  # type: ignore[assignment]
         else:
             kwargs["encoding"] = "utf8"
         value_ = etree.Element(
             "{%s}text" % xml_namespace_map[None],
-            **kwargs,  # type: ignore
+            **kwargs,  # type: ignore[arg-type]
```

Comment:
Changing 1 line at the bottom of this file triggered pre-commit-hook, whose dogs barked at me that ignore-comments should always have an exact error code they are ignoring :-)